packages/components/src/interfaces/complex/DynamicStatistics.ts

```diff
@@ -0,0 +1,85 @@
+import type { Static } from "@sinclair/typebox"
+import { Type } from "@sinclair/typebox"
+
+import type { IsomerSiteProps, LinkComponentType } from "~/types"
+
+export const DYNAMIC_STATISTICS_TYPE = "dynamicstatistics"
+
+// Hardcoded for now because
+// 1. MUIS is the only use case and there's always 6 prayer timeslots
+// 2. No other known use cases have been identified
+export const NUMBER_OF_STATISTICS = 6
```

Comment:
technically we can infer it from the received response of the json. but agree we don't have any use case for now and our UI is not generic enough as well. can be a future consideration.